pkg/virt-config/virt-config.go

```diff
@@ -101,6 +106,64 @@ func IsPPC64(arch string) bool {
 	return arch == "ppc64le"
 }
 
+func (c *ClusterConfig) GetCpuModelToFeaturesToDisableMap(cpuModelConfigMapLister corev1.ConfigMapNamespaceLister) map[string]*cmdv1.FeaturesToDisable {
+	/*
+					Libvirt validation fails when a CPU model is usable
+					by QEMU but lacks features listed in
+					`/usr/share/libvirt/cpu_map/[CPU Model].xml` on a node
+
+					To avoid the validation error mentioned above we will disable
+					some of the features in the `/usr/share/libvirt/cpu_map/[CPU Model].xml` files
+					for some of the cpu models.
+					Examples of validation error:
+		    		https://bugzilla.redhat.com/show_bug.cgi?id=2122283
+					https://gitlab.com/libvirt/libvirt/-/issues/304
+
+					Additionally, to handle future issues, we allow setting a config map named
+					“cpu-model-to-features-to-disable” in the installation namespace.
+					This enables quick and targeted disabling of problematic features.
+
+					Issue in Libvirt: https://gitlab.com/libvirt/libvirt/-/issues/608
+					once the issue is resolved we can remove the key,value pairs with the mpx feature
+	*/
+
+	cpuModelToFeaturesToDisable := map[string]*cmdv1.FeaturesToDisable{
```

Comment:
Hey Vasiliy,

Just to let you know, MPX is outdated and turned off automatically, even if it's listed in a node or the cpu_map file. You can find more details at https://gitlab.com/libvirt/libvirt/-/issues/608. So, nobody will be surprised by this :)

About SVM with Opteron_G2, I'll go with Libvirt's suggestion and classify it under obsolete CPU models since it's very old and we shouldn't continue supporting it. I've already made a commit for this change, and I'll also remove it from the hardcoded map.